Scripts/lem.py

```diff
@@ -39,15 +41,15 @@ def main(args):
         }
     }
     # Check input data folders/files exist
-    if not os.path.exists(base_zonedata_path):
+    if not base_zonedata_path.exists():
```

Comment:
These checks could be is_dir() instead of exists() because we expect the paths to be directories later. Not strictly required, but it might help to catch some incorrect arguments.